wagtailio/project_styleguide/templates/patterns/includes/header.html

```diff
@@ -1,11 +1,18 @@
 <header class="header">
-    <a class="header__link" href="/" aria-label="Wagtail.org homepage">
-        <svg class="header__logo" aria-hidden="true">
-            <use xlink:href="#wagtail" />
-        </svg>
-    </a>
+    <div class="header__col-one">
+        <a class="header__link" href="/" aria-label="Wagtail.org homepage">
+            <svg class="header__logo" aria-hidden="true">
+                <use xlink:href="#wagtail" />
+            </svg>
+        </a>
+
+        <div class="header__github-button">
+            <a class="github-button" href="https://github.com/wagtail/wagtail" data-show-count="true" aria-label="Star Wagtail on GitHub">Star</a>
```

Comment:
Question: This just links straight to the wagtail Github, does the aria-label make sense to screen readers here?

Reply:
It was provided by Github but I agree that it needs context. I'll add